### ISSUE TITLE
iOS14 - Pause/Play button may not work with VoiceOver enabled

### ISSUE THREAD

**Issue by terracoda** [phetsims/molecules-and-light#376]
iOS 14 was just coming out when MAL was published, so it may not have been fully tested with VoiceOver.

I just heard that the Pause/Play button was not working for a VO user who was using iOS 14.

@jessegreenberg volunteered to check out this issue.

**Comment by terracoda** [comment]
@KatieWoe said she could chime in and help if needed.

**Comment by KatieWoe** [comment]
I confirmed the issue with the play pause button. Even more worryingly, I wasn't able to turn on the light source. I also couldn't press the mute button. I was able to change speed, molecule, and source, but buttons may be the issue. When trying to press them, I heard the sounds that I would expect, but nothing happened.

I tested another sim (Gravity Force Lab) and saw the same sort of issue. I was unable to use the Constant Size checkbox and I was unable to "double tap and hold" to move the ruler. This indicates a more general problem than just MaL.

**Comment by jessegreenberg** [comment]
I just upgraded to iOS 14, and I am noticing some other differences with this upgrade. But buttons including the PlayPauseButton and ResetAllButton are working as I would expect with VoiceOver.

**Comment by KatieWoe** [comment]
Did a quick check on Mac 10.15.7 on Safari 14 and was able to use the buttons there.

**Comment by jessegreenberg** [comment]
OK, I just tested the published version and I did see the problem. So this is NOT a problem on master, but is a problem on the published version. It looks like everything is getting two events, so the play/pause button is getting fired twice rapidly which explains why @KatieWoe heard the sound but it didn't change state.

**Comment by KatieWoe** [comment]
That explains why I saw the problem in some cases, but not others.

**Comment by jessegreenberg** [comment]
Indeed, this is a general issue. It looks like many things are getting both pointer events and PDOM click events now so things are getting the PDOM click event and the pointer event. Moving to scenery.
